skorch/net.py

```diff
@@ -612,8 +629,10 @@ def predict_proba(self, X):
         y_proba : numpy ndarray
 
         """
-        y_proba = self.forward(X, training=False)
-        y_proba = to_numpy(y_proba)
+        y_probas = []
+        for yp in self.forward_iter(X, training=False):
+            y_probas.append(to_numpy(yp))
```

Comment:
Why not a list comprehension?

Reply:
`y_probas = [to_numpy(yp) for yp in self.forward_iter(X, training=False)]` -> 80 chars with indents.